### ISSUE TITLE
Card Overview: Filter

### ISSUE THREAD

**Issue by Dubidubiduu** [nextcloud/deck#2232]
**Is your feature request related to a problem? Please describe.**
@jakobroehrl asked a question on moving cards at the new overview-board in #2231. The screenshot in this issue doesn't show a filter or is it just not in the picture? If so, in real use, with tons of boards, this could get messy.

**Describe the solution you'd like**
If it is not already planned, I really would love to see the filter option of regular boards here, just without due-stuff I guess. So I could see only my stuff, filter the cards tailored others(!) and I'm also able to filter by tag (e.x. important).

The overview looks very promising thou and will make my workflow much more efficient! Thank you!!

**Comment by juliushaertl** [comment]
The view will be filtered already to only show cards that:
- have a due date
- are assigned to you OR are on a board that is only available to you

We are of course open to suggestions for further useful filtered views, but I think this is a good starting point for showing only cards that are relevant to the user.

**Comment by Dubidubiduu** [comment]
> We are of course open to suggestions for further useful filtered views, but I think this is a good starting point for showing only cards that are relevant to the user.

Yeah, there need to be a start. In perspective I really want to make my case strong: When you organize a bunch of people it would be really helpful to know what someone else's workload is, without checking every board.
